pages/Real-world usage.md

```diff
@@ -61,17 +61,24 @@ For example, we can start five Redix processes under our supervision tree and
 name them `:redix_0` to `:redix_4`:
 
 ```elixir
+# lib/my_app.ex
+
+# create the redix children list of workers
 pool_size = 5
-children = for 0..(pool_size - 1)
+redix_children = for i <- 0..(pool_size - 1) do
   worker(Redix, [[], [name: :"redix_#{i}"]], id: {Redix, i})
 end
+# add them to the supervisior children list
+children = children ++ redix_children
```

Comment:
Let's remove this. We don't know what the user wants to do with this list of Redix workers, maybe they want to supervise them with a separate supervisor, maybe like this.